jax/core.py

```diff
@@ -58,6 +58,18 @@ def __str__(self):
     return str(pp_jaxpr(self))
   __repr__ = __str__
 
+  def subjaxprs(self):
```

Comment:
Maybe make this a top-level function rather than a method? That seems more consistent with the existing style of the file (since right now Jaxpr is just a struct class).

Reply:
I have done this, but reluctantly. The current style is itself inconsistent: the Jaxpr class does not have methods, but other classes do have them. Is there a rationale? Is it attribute lookup performance that worries you? I think that careful use of methods vs. global methods can help modularise the code and the namespaces.

Reply:
I think it's consistent: some classes act like simple structs, others don't. This one acts like a struct.

I raised it only as a consistency issue. The only performance implication I can think of is that it may be easier someday to push simple structs and top-level functions into low-level code one at a time, rather than having to deal with classes.